pkg/ansible/runner/fake/runner.go

```diff
@@ -28,6 +28,7 @@ type Runner struct {
 	Finalizer                   string
 	ReconcilePeriod             time.Duration
 	ManageStatus                bool
+	AnsibleDebugLogs            bool
```

Comment:
I don't think I'd add this argument to the watches.yaml, I'd make it a cli flag/environment variable, since I assume this is something users would want to set after their operator is deployed in the cluster rather than at container build time

Reply:
Make sense.